sdk/python/kfp/client/client.py

```diff
@@ -517,79 +513,78 @@ def list_experiments(
         page_size: int = 10,
         sort_by: str = '',
         namespace: Optional[str] = None,
-        filter: Optional[str] = None
-    ) -> kfp_server_api.ApiListExperimentsResponse:
+        filter: Optional[str] = None,
+    ) -> kfp_server_api.V2beta1ListExperimentsResponse:
         """Lists experiments.
 
         Args:
             page_token: Page token for obtaining page from paginated response.
             page_size: Size of the page.
-            sort_by: Sort string of format ``'[field_name]', '[field_name] desc'``. For example, ``'name desc'``.
+            sort_by: Sort string of format ``'[field_name]', '[field_name] desc'``. For example, ``'display_name desc'``.
             namespace: Kubernetes namespace to use. Used for multi-user deployments. For single-user deployments, this should be left as ``None``.
             filter: A url-encoded, JSON-serialized Filter protocol buffer
-                (see `filter.proto message <https://github.com/kubeflow/pipelines/blob/777c98153daf3dfae82730e14ff37bdddc334c4d/backend/api/filter.proto>`_). For a list of all filter operations ``'op'``, see `here <https://github.com/kubeflow/pipelines/blob/777c98153daf3dfae82730e14ff37bdddc334c4d/sdk/python/kfp/client/client.py#L37-L45>`_. Example:
+                (see `filter.proto message <https://github.com/kubeflow/pipelines/blob/cb7d9a87c999eb1d2280959e5afbeee9e270ef3d/backend/api/v2beta1/filter.proto>`_). Example:
 
                   ::
 
                     json.dumps({
                         "predicates": [{
-                            "op": _FILTER_OPERATIONS["EQUALS"],
-                            "key": "name",
+                            "operation": "EQUALS",
```

Comment:
Curious: why remove `_FILTER_OPERATIONS` here?

Reply:
This is in the docstring part, and `_FILTER_OPERATIONS` isn't exposed, the example is not really runnable--it needs to be at least `kfp.client._FILTER_PERATIONS[...]` to be runnable. 
Also the API works with either the enum name or the enum value, if users has to know the key `EQUALS`, there's really no point for them to convert the key to its enum value here.